Makefile

```diff
@@ -28,30 +35,30 @@ ROOTCFLAGS := $(shell root-config --cflags)
 ROOTLIBS   := $(shell root-config --libs --evelibs --glibs)
 CFLAGS   = \
 	-fPIC -w -g -W ${ROOTCFLAGS} \
-	-Wfatal-errors -Wall -Wextra -Wpedantic -Wconversion -Wshadow
+	-Wfatal-errors -Wall
 LFLAGS   = ${ROOTLIBS} -lTreePlayer -g -lGenVector \
 	-lRooFit -lRooFitCore -lRooStats -lMinuit
 DEPFLAGS = -MT $@ -MMD -MP -MF $(dirDEP)/$*.Td
 POSTCOMPILE = @mv -f $(dirDEP)/$*.Td $(dirDEP)/$*.d && touch $@
 
 
 # * INVENTORIES OF FILES * #
-SOURCES = $(wildcard ${dirSOURCES}/*.cpp) $(wildcard ${dirSOURCES}/**/*.cpp)
-SCRIPTS = $(wildcard ${dirSCRIPTS}/*.C) $(wildcard ${dirSCRIPTS}/**/*.C)
+SOURCES = $(wildcard ${dirSOURCES}/*.${extSOURCE}) $(wildcard ${dirSOURCES}/**/*.${extSOURCE})
+SCRIPTS = $(wildcard ${dirSCRIPTS}/*.${extSCRIPT}) $(wildcard ${dirSCRIPTS}/**/*.${extSCRIPT})
```

Comment:
Just a small question here. You use two '*' to denote the name of the subdirectory. Is only one '*' is sufficient?

Reply:
The double `**` is to include subdirectories. They are not (yet) there, but is for later purposes.

If I remember correctly, `$(wildcard ${dirSCRIPTS}/**.C)` does not work, but you can try.

Reply:
OK. Thanks for the reply.